src/lsrs/helper/tlp-transport.ts

```diff
@@ -0,0 +1,109 @@
+import axios from "axios";
+import {ILogObject, TTransportLogger} from "tslog/src/interfaces";
+
+/**
+ * Domain class representing an error object in the log.
+ */
+class ErrorLog {
+
+    private readonly className: string;
+    private readonly message: string;
+    private readonly stackTrace?: string;
+
+    constructor(error: Error) {
+        this.className = error.constructor.name;
+        this.message = error.message;
+        this.stackTrace = typeof error.stack == "string"
+            ? error.stack
+            : JSON.stringify(error.stack);
+    }
+}
+
+/**
+ * Domain class representing a TLP API compatible log message.
+ */
+class TLPLogMessage {
+
+    private static readonly _SOURCE_APPLICATION = "lsrs";
+
+    private readonly source: string;
+    private readonly threadName: string;
+    private readonly timeStamp: number;
+    private readonly loggerName: string;
+    private readonly level: {levelStr: string};
+    private readonly content: string;
+    private readonly exception?: ErrorLog;
+    
+    /**
+     * This constructor extract and converts log entry data from a TSLog log entry object.
```

Comment:
typo: extract**s**

Reply:
That would be the only change now, so I'll add it to the next PR, thanks!